packages/conf-libssl/conf-libssl.3/files/homebrew.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -ex
+
```

Comment:
It would make sense to add a line of comment what this script does, e.g.:

`# symlink the homebrew pkg-config files for openssl to the opam local pkg-config library`

Reply:
done. Thanks!